juju/bundle.py

```diff
@@ -432,8 +430,16 @@ def get_charm_series(path):
             mark = exc.problem_mark
             log.error("Error parsing YAML file {}, line {}, column: {}".format(md, mark.line, mark.column))
         raise
-    series = data.get('series')
-    return series[0] if series else None
+    _series = data.get('series')
+    series = _series[0] if _series else None
+
+    if series is None:
+        model_config = await model.get_config()
+        default_series = model_config.get("default-series")
```

Comment:
In my case I have a k8s charm, but the "default-series" it has stored is:
`<ConfigValue source='default' value='focal'>`
Is that normal?

Reply:
Yeah that's normal, that's why we get the `.value` out of it (#L440), but we need to check if it's `None` first, though maybe I can make the coding a bit clearer there.